packages/core/CHANGELOG.md

```diff
@@ -14,10 +14,6 @@ All notable changes to the Zowe core SDK package will be documented in this file
 
 - BugFix: Updated dependencies to resolve problems with the ansi-regex package
 
-## `6.33.1`
-
-- Migrated from TSLint (now deprecated) to ESLint for static code analysis.
-
```

Comment:
Why was this removed? Same question applies to the changelogs below.

Reply:
IIRC we made the decision to not document dev-only releases in the changelog.

Reply:
Generally yes, but IIRC we published the patch release 6.33.1 for no other reason than migrating from TSLint to ESLint, and agreed to add the line to the changelog then, so I'd be hesitant to remove it now.

Reply:
Makes sense 👍 
I'll ad it back : )